hack/build/docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:27
```

Comment:
why fedora 27 here and f28 elsewhere?

Reply:
Ah, I think I saw that kubevirt was using 27.  I'll unify it throughout.  The controller and importer images are yet to get overhauled.